RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -64,53 +65,41 @@ namespace lst::t5dnn {
-        float(modulesInGPU.layers[lowerModuleIndex3] + 6 * is_endcap3),  // outer T3 anchor hit 3 layer (t3_0_layer)
         mdsInGPU.anchorEta[mdIndex4],                                    // outer T3 anchor hit 4 eta (t3_2_eta)
-        mdsInGPU.anchorPhi[mdIndex4],                                    // outer T3 anchor hit 4 phi (t3_2_phi)
-        mdsInGPU.anchorZ[mdIndex4],                                      // outer T3 anchor hit 4 z (t3_2_z)
-        alpaka::math::sqrt(acc, x4 * x4 + y4 * y4),                      // outer T3 anchor hit 4 r (t3_2_r)
+        mdsInGPU.anchorZ[mdIndex4] / t5dnn::Z_max,                       // outer T3 anchor hit 4 z (t3_2_z)
+        alpaka::math::sqrt(acc, x4 * x4 + y4 * y4) / t5dnn::R_max,       // outer T3 anchor hit 4 r (t3_2_r)
         float(modulesInGPU.layers[lowerModuleIndex4] + 6 * is_endcap4),  // outer T3 anchor hit 4 layer (t3_2_layer)
         mdsInGPU.anchorEta[mdIndex5],                                    // outer T3 anchor hit 5 eta (t3_4_eta)
-        mdsInGPU.anchorPhi[mdIndex5],                                    // outer T3 anchor hit 5 phi (t3_4_phi)
-        mdsInGPU.anchorZ[mdIndex5],                                      // outer T3 anchor hit 5 z (t3_4_z)
-        alpaka::math::sqrt(acc, x5 * x5 + y5 * y5),                      // outer T3 anchor hit 5 r (t3_4_r)
+        mdsInGPU.anchorZ[mdIndex5] / t5dnn::Z_max,                       // outer T3 anchor hit 5 z (t3_4_z)
+        alpaka::math::sqrt(acc, x5 * x5 + y5 * y5) / t5dnn::R_max,       // outer T3 anchor hit 5 r (t3_4_r)
         float(modulesInGPU.layers[lowerModuleIndex5] + 6 * is_endcap5),  // outer T3 anchor hit 5 layer (t3_4_layer)
-        alpaka::math::log10(acc, (innerRadius + outerRadius) * lst::k2Rinv1GeVf),  // T5 pT (t5_pt)
-        mdsInGPU.anchorEta[md_idx_for_t5_eta_phi],                                 // T5 eta (t5_eta)
-        mdsInGPU.anchorPhi[md_idx_for_t5_eta_phi],                                 // T5 phi (t5_phi)
-        alpaka::math::log10(acc, innerRadius),                                     // T5 inner radius (t5_innerRadius)
-        alpaka::math::log10(acc, bridgeRadius),                                    // T5 bridge radius (t5_bridgeRadius)
-        alpaka::math::log10(acc, outerRadius)                                      // T5 outer radius (t5_outerRadius)
+        t5_eta,                                                          // T5 eta (t5_eta)
+        alpaka::math::log10(acc, innerRadius),                           // T5 inner radius (t5_innerRadius)
+        alpaka::math::log10(acc, bridgeRadius),                          // T5 bridge radius (t5_bridgeRadius)
+        alpaka::math::log10(acc, outerRadius)                            // T5 outer radius (t5_outerRadius)
     };
 
-    // (0): Linear(in_features=38, out_features=32, bias=True) => x = x*W_T + b
+    // (0): Linear(in_features=24, out_features=32, bias=True) => x = x*W_T + b
     float x_0[32];
```

Comment:
may want to define `unsigned int kOutFeatures = 32;` to avoid repetition and have a faster way to update if ever decide to change